pkg/utils/cmd/util.go

```diff
@@ -41,3 +42,36 @@ func GetUserSetVar(cmd *cobra.Command, flagName, envKey string, isOptional bool)
 	return "", errors.New("Neither " + flagName + " (command line flag) nor " + envKey +
 		" (environment variable) have been set.")
 }
+
+// GetUserSetVarFromArrayString returns values either command line flag or environment variable
+func GetUserSetVarFromArrayString(cmd *cobra.Command, flagName, envKey string, isOptional bool) ([]string, error) {
```

Comment:
GetUserSetVarFromArrayString is almost the same as GetUserSetVarFromString, but I am not 100% sure if it is worth consolidating. The problem is that this function will be copied to all projects. Maybe we should put them in core. Any ideas?

Reply:
I will add it in next PR